mimic/canned_responses/maas_test_check.py

```diff
@@ -0,0 +1,67 @@
+"""
+Canned response for /test-check
+"""
+
+import collections
+
+
+def test_check(checkType):
```

Comment:
`checkType` ought to be `check_type` for coding-standard compliance; Mimic uses PEP 8 naming conventions.